### ISSUE TITLE
RNN example in tutorial is broken

### ISSUE THREAD

**Issue by udibr** [pfnet/chainer#218]
I had to replace

    h = Variable(np.zeros((50,), dtype=np.float32), volatile=volatile)
    loss = 0

with

    N = x_list.shape[1]  # minibatch size
    h = Variable(np.zeros((N,50), dtype=np.float32), volatile=volatile)
    loss = Variable(np.zeros(N))

**Comment by unnonouno** [comment]
Thank you for reporting mistake.

Yes, shape of `h` is incorrect. As you say, it depends on the shape of `x_list`. It is simpler one:
```
h = Variable(np.zeros((1, 50), dtype=np.float32), volatile=volatile)
```

I thinks `loss = 0` is valid because `softmax_cross_entropy` returns a Variable of 0-dimension.

**Comment by udibr** [comment]
you are right about `loss = 0` (sorry)
